src/fondant/pipeline/runner.py

```diff
@@ -78,6 +80,24 @@ def run(
         else:
             self._run(input)
 
+    @staticmethod
+    def check_docker_installation():
+        try:
+            # Check Docker info
+            subprocess.call(["docker", "info"])  # nosec
+
+            # Check Docker Compose version
+            subprocess.call(["docker", "compose", "version"])  # nosec
+
+        except subprocess.CalledProcessError:
+            msg = (
+                "Docker or Docker Compose is not installed or not running. Please make sure "
```

Comment:
Can we maybe separate the errors for each to be more specific on what needs to be installed?